source/agora/utils/InetUtils.d

```diff
@@ -174,6 +190,145 @@ version (Windows)
     }
 }
 
+    /***************************************************************************
+        Expand an IPv6 address to its canonical representation
```

Comment:
Space

Reply:
for the 'getImportPaths': I was wondering if the reason for missing stdx is that in dub.json the submodule/stdx-allocator is not referenced.

Reply:
Still missing the newline